### DISCUSSION TITLE
Acquiring Datasets

### DISCUSSION THREAD

**Discussion by pmd84** [microsoft/Data-Science-For-Beginners#160] [Q&A] [answered]
Hi! I am enjoying the content and structure of this course so far. One issue I am running into is accessing the data used for the practice and assignments. In Section 1 (Introduction) Lesson 4 (Stats and Probability), I am expected to have two sets of data - the MLB Heights and Weights as well as the Diabetes data. I am given links to the websites from which data was pulled, but I am given no instructions on how to pull data from tables on webpages. I think it would be incredibly helpful if the .tsv files were uploaded along with the lesson for downloading and using.

Thank you.

**Comment by jlooper** [comment]
Great feedback! Let me see what I can do :)

**Comment by jlooper** [comment, marked as answer]
aha, I forgot, the data is stored in the root /data folder :) - you'll find the datasets you need there. thanks!
